### PR TITLE
chore: support to use kustomize install all resources

### PR DESCRIPTION
**Pull request by tokers**
Closing #54 .

### CODE REVIEW

**Comment by membphis** [comment]
love this PR, need @gxthrj confirm

**Review by gxthrj — APPROVED** [review]

**Comment by moonming** [comment]
please remove Chinese doc

**Comment by tokers** [comment]
> please remove Chinese doc

@gxthrj The deploy.md is written in Chinese, is this deliberate?

**Comment by tokers** [comment]
Contents in deploy.md is highly duplicated with others, maybe we can just remove it.

**Comment by gxthrj** [comment]
yes， we can remove it ,  and the content is moved to `usage.md` .

**Comment by tokers** [comment]
> please remove Chinese doc

Removed.

**Comment by membphis** [comment]
@tokers you delete file `docs/deploy.md`, I think we need to update some documents that link to this file.

File: docs/develop.md [hunk 1]
```diff
@@ -36,65 +36,8 @@ Configure the service address of Apache APISIX to conf/apisix/base_url ($GOPATH/
...
-  scope: Namespaced
-  names:
-    plural: apisixupstreams
-    singular: apisixupstream
-    kind: ApisixUpstream
-    shortNames:
-    - au
-
-EOF
+```
```
Comment (gxthrj) [inline docs/develop.md:39]: add `shell`

**Comment by tokers** [comment]
> @tokers you delete file `docs/deploy.md`, I think we need to update some documents that link to this file.

I have look through files in this project, it seems no file references to deploy.md.

**Comment by tokers** [comment]
Fixed.

**Review by membphis — COMMENTED** [review]
LGTM

File: docs/develop.md (hunk 1 above)
Comment (tokers) [reply docs/develop.md]: Fixed.

**Pull request merged**